sdk/storagev2_model.go

```diff
@@ -0,0 +1,253 @@
+// Copyright 2021 ProximaX Limited. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package sdk
+
+import (
+	"fmt"
+)
+
+type DataModificationState uint8
+
+const (
+	Succeeded DataModificationState = iota
+	Cancelled
+)
+
+type ActiveDataModification struct {
+	Id                 *Hash
+	Owner              *PublicAccount
+	DownloadDataCdi    *Hash
+	ExpectedUploadSize StorageSize
+	ActualUploadSize   StorageSize
+	FolderName         string
+	ReadyForApproval   bool
+}
+
+func (active *ActiveDataModification) String() string {
+	return fmt.Sprintf(
+		`
+			"Id": %s,
+			"Owner": %s,
+			"DownloadDataCdi": %s,
+			"ExpectedUploadSize": %d,
+			"ActualUploadSize": %d,
+			"FolderName": %s,
+			"ReadyForApproval": %t, 
+		`,
+		active.Id.String(),
+		active.Owner.String(),
+		active.DownloadDataCdi.String(),
+		active.ExpectedUploadSize,
+		active.ActualUploadSize,
+		active.FolderName,
+		active.ReadyForApproval,
+	)
+}
+
+type CompletedDataModification struct {
+	ActiveDataModification []*ActiveDataModification
+	State                  DataModificationState
+}
+
+func (completed *CompletedDataModification) String() string {
+	return fmt.Sprintf(
+		`
+			"ActiveDataModification": %s,
+			"State:" %d,
+		`,
+		completed.ActiveDataModification,
+		completed.State,
+	)
+}
+
+type ConfirmedUsedSize struct {
+	Replicator *Hash
+	Size       StorageSize
+}
+
+func (confirmed *ConfirmedUsedSize) String() string {
+	return fmt.Sprintf(
+		`
+			"Replicator": %s,
+			"Size:" %d,
+		`,
+		confirmed.Replicator,
+		confirmed.Size,
+	)
+}
+
+type VerificationState uint8
+
+const (
+	PendingVerification VerificationState = iota
+	CanceledVerification
+	FinishedVerification
+)
+
+type VerificationOpinion struct {
+	Prover *Hash
+	Result uint8
+}
```

Comment:
Did you do it? There is a PR with `EndDriveVerification` https://github.com/proximax-storage/go-xpx-chain-sdk/pull/110

Reply:
Yes, I added it when I asked you about it on 10/Dec/2021 on Slack. I wasn't aware of the PR. You can replace this code with it

Reply:
OK. will be replaced by https://github.com/proximax-storage/go-xpx-chain-sdk/pull